docker/rococo-node.dockerfile

```
@@ -0,0 +1,44 @@
+# Requires to run from repository root and to copy the binary in the build folder
+FROM --platform=linux/amd64 ubuntu:focal AS base
+LABEL maintainer="Frequency Team"
+LABEL description="Frequency parachain full node for Rococo testnet"
+
+RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
```

You can save some disk space with adding this at the end.

```suggestion
RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates && apt-get clean
```